pages/pricing/_pricing-iac-library.html

```diff
@@ -0,0 +1,80 @@
+<ul>
+  <li class="grid-container text-bold text-white">
+    <span
+      class="light"
+      title=""
```

Comment:
One idea: extract the contents of these rows into a YAML file (or multiple files, one per section of the pricing table) in the `_data` folder. That'll make it easier to read and for me to fill in details, and then instead of lots of copy/pasted HTML markup here, we'd have a DRY for-loop over that data.

Reply:
I created the file pricing-table.yml, where you can change all the content needed. For the rows adds-on and pricing you can edit the content in files _pricing-add-ons and _pricing.

Reply:
I'm not seeing `pricing-table.yml` in the files changed of this PR? Did you forget to push some code by any chance?

Reply:
It is already there.